crates/rspack_core/src/dependency/mod.rs

```diff
@@ -208,12 +209,19 @@ impl ExportSpec {
   }
 }
 
-#[derive(Debug, Default)]
+#[derive(Debug)]
+pub enum ExportSpecItem {
+  True,
+  String(Box<Vec<JsWord>>),
+  Object(Box<ExportSpec>),
+}
+
+#[derive(Debug)]
 pub struct ExportsSpec {
```

Comment:
Here is conflicted.

Reply:
![image](https://github.com/web-infra-dev/rspack/assets/17974631/f8faa537-8815-4195-b7d1-c4f4c0c3f8a9)

Reply:
here is webpack data structure.